<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up_window</class>
 <widget class="QMainWindow" name="Sign_up_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>374</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_Inputer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_Inputer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="Teacher_QRadio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>老师</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Student_QRadio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sign_up_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_Inputer_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>重复密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:100%'&gt;密码在6-18位之间，&lt;br&gt;至少包含一个大写字母、一个小写字母和一个数字。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_Inputer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>职工号/学号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background_image</zorder>
   <zorder>label</zorder>
   <zorder>account_Inputer</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Password_Inputer</zorder>
   <zorder>Teacher_QRadio</zorder>
   <zorder>Student_QRadio</zorder>
   <zorder>Sign_up_button</zorder>
   <zorder>Password_Inputer_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>number_Inputer</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Teacher_QRadio</tabstop>
  <tabstop>Student_QRadio</tabstop>
  <tabstop>account_Inputer</tabstop>
  <tabstop>number_Inputer</tabstop>
  <tabstop>Password_Inputer</tabstop>
  <tabstop>Password_Inputer_2</tabstop>
  <tabstop>Sign_up_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
